src/Text/Ogmarkup/Private/Ast.hs

```diff
@@ -15,6 +15,7 @@ data Component a =
     Teller [Format a]            -- ^ A narrative description
   | Dialogue (Reply a) (Maybe a) -- ^ A dialogue reply
   | Thought (Reply a) (Maybe a)  -- ^ Inner dialogue of the character.
+  | Whatever a
```

Comment:
Maybe you can add a piece of documentation (just a few words) here. Just to say the purpose. Also, I would have named it `IllFormed`. What do you think?